config.go

```diff
@@ -84,21 +30,21 @@ type ServiceRegistrationConfig struct {
 
 // ServiceRegistration represents the parsed json data from a ServiceRegistrationConfig
 type ServiceRegistration struct {
```

Comment:
Added JSON tags so that we can save expectations in a file rather than having to write everything out line by line which is extremely tedious and can lead to our tests being incomplete (e.g. someone added a `len(thing)` check but didn't add a `thing[0] == X` check. 

By comparing the entire file and unmarshaling it to the actual struct we get more accurate and easy to maintain tests.